docs/admin-manual/singularity-support.rst

```diff
@@ -1,20 +1,19 @@
-Singularity Support
-===================
+Apptainer/Singularity Support
+=============================
 
 :index:`Singularity<single: Singularity; installation>` :index:`Singularity`
 
 Singularity (https://sylabs.io/singularity/) is a container runtime system
-popular in scientific and HPC communities.  Apptainer is an open
-source fork of Singularity that is API and CLI compatible with
-singularity.  Everything in this document that pertains to
-Singularity also is true for the Apptainer container runtime.
-HTCondor can run jobs
-inside Singularity containers either in a transparent way, where the
-job does not know that it is being contained, or, the HTCondor
-administrator can configure the HTCondor startd so that a job can
-opt into running inside a container.  This allows the operating
-system that the job sees to be different than the one on the host system,
-and provides more isolation between processes running in one job and another.
+popular in scientific and HPC communities.  Apptainer (https://apptainer.org)
```

Comment:
Should the link be an actual :ref: that users can click on?

Reply:
I inspected the built documentation and didn't realize that raw links actually created hrefs so disregard this comment